.github/workflows/checks.yaml

```diff
@@ -60,6 +59,17 @@ jobs:
         working-directory: ${{ inputs.package-path }}
         run: dart pub global run dependency_validator
 
+  validate-publish:
+    runs-on: ubuntu-latest
+    # only run on release pull requests
+    if: ${{ startsWith(github.ref, 'refs/head/release') && github.event.sender == 'rmconsole-readonly-wk' }}
```

Comment:
Is it a guarantee that the sender will be `rmconsole-readonly-wk`? I thought we had multiple service accounts and that any of them could be involved based on rate limiting availability

Reply:
Looks like we have 9 of them: https://github.com/orgs/Workiva/people?query=rmconsole

I guess I thought that `readonly` was the one that was used for release prs, although, that might be invalid

We could update that to `contains(github.event.sender, 'rmconsole')`, probably would be a better check

Reply:
Maybe we can find a different/better way to identify release PRs? Technically, it's possible and totally valid from RM's perspective to manually create a release PR - as long as it meets a couple of criteria, RM will recognize it as such even if it didn't create the PR.

Reply:
Well, from the perspective of gha-dart-oss, we want to validate for any PR that results in a tag

I'm not sure we'd be able to know that without hitting some, probably non-existant, rmconsole api

An alternate idea would be any PR that changes `pubspec.yaml#version`? would require a bit of introspection into the PR in question though